pkg/controllers/csi/metadata/sqlite_gorm_client.go

```diff
@@ -54,14 +63,31 @@ func NewDBAccess(path string) (*DBConn, error) {
 	db, err := gorm.Open(sqlite.Open(path), &gorm.Config{Logger: logger.Default})
```

Comment:
Please remove this debug logger, otherwise we print every SQL on production.

Reply:
Fixed [here](https://github.com/Dynatrace/dynatrace-operator/pull/2906/commits/9f878cf24ec2d98fea11ce6cb741d0ad01577d0b)